Musician Module/Controls/Instrument/Flute/FlutePreview.cs

```diff
@@ -22,6 +22,7 @@ public void Press(GuildWarsControls key)
                 case GuildWarsControls.HealingSkill:
                 case GuildWarsControls.UtilitySkill1:
                 case GuildWarsControls.UtilitySkill2:
+                    AudioPlaybackEngine.Instance.StopSound();
```

Comment:
each instrument needs its own preview class in case instrument skills differ.
This is a false-positive. It really should stay.